espnet2/train/preprocessor.py

```diff
@@ -1340,3 +1340,41 @@ def __call__(
         data = super()._speech_process(data)
         data = self._speech_process(uid, data)
         return data
+
+
+class DiarPreprocessor(CommonPreprocessor):
```

Comment:
I think you can just use `AbsPreprocessor` as the super class, since no method in `CommonPreprocessor` is used here.